vm/src/circuit/r1cs.rs

```diff
@@ -190,24 +174,8 @@ impl R1CS {
         j
     }
 
-    pub fn not(&mut self, output: &str, input: &str) -> usize {
-        let i = self.var(output);
-        let j = self.var(input);
-        self.constraint(|_cs, a, b, c| {
-            a[0] = ONE;
-            a[j] = MINUS;
-            b[0] = ONE;
-            c[i] = ONE;
-        });
-        i
-    }
-
-    pub fn set_not(&mut self, output: &str, input: &str) -> usize {
-        self.set_field_var(output, ONE - self.get_var(input));
-        self.not(output, input)
-    }
-
-    pub fn to_bits(&mut self, name: &str, val: u32) -> usize {
+    #[allow(clippy::wrong_self_convention)]
+    pub fn to_bits(&mut self, name: &str, val: u32) {
```

Comment:
This used to return a `usize`, specifically the `vj` value.

I admittedly don't follow this part yet, but am curious: is the output perhaps still needed or do the side-effects of calling this mutable method suffice?

Reply:
Side-effects suffice, as the constraint system modifies itself.